system/metrics/sinks/elastic/elasticSink.cpp

```diff
@@ -175,9 +177,16 @@ bool ElasticMetricSink::getDynamicMappingSuffixesFromIndex(const IPropertyTree *
     std::string endpoint;
     endpoint.append("/").append(indexName.str()).append("/_mapping");
     httplib::Result res = pClient->Get(endpoint.c_str(), elasticHeaders);
+
+    if (res == nullptr)
```

Comment:
per online search:
Error Handling:
If the Result is null, it indicates that the request failed. You can use httplib::Error enum to get more details about the error. 
and I found this sample error handling pattern:
 if (res && res->status == 200)
    {
        std::cout << res->body << "\n";
    }
    else
    {
        auto err = res.error();
        std::cout << httplib::to_string(err) << "\n";
    }